core/src/main/resources/schema/app/20181111021057_BUG-114205_stack_datalake_field.sql

```diff
@@ -0,0 +1,9 @@
+-- // BUG-114205 stack datalake field
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE stack ADD COLUMN IF NOT EXISTS datalake BOOLEAN DEFAULT FALSE;
```

Comment:
I think it should be better to using an ENUM like type

Reply:
Why?